client/src/index.css

```diff
@@ -227,6 +227,6 @@ p a, p a:hover {
     top: 0;
     right: 0;
     height: auto;
-    width: 30%;
+    width: auto;
```

Comment:
why? i liked the physics panel more when it was a constant width

Reply:
It took up too much space.  I'd prefer even a little square in the corner with a wrench icon or something when it's not in use.

Reply:
ok. let me add this to TODOs